internal/pipe/nfpm/nfpm.go

```diff
@@ -154,7 +154,7 @@ func create(ctx *context.Context, format, arch string, binaries []artifact.Artif
 	}
 
 	var path = filepath.Join(ctx.Config.Dist, name+"."+format)
-	log.WithField("file", path).Info("creating")
+	log.WithField("file", path)
```

Comment:
this one we can leave as it was before:

```suggestion
	log.WithField("file", path).Info("creating")
```